### PR TITLE
fix grammar, typo in some md files

### PR DESCRIPTION
**Pull request by cs5055**
### Description

Correction to live-queries markdown file. Fixing typos, links and punctuations.
### Affected components 

- Server
- Console
- CLI
- Docs
- Community Content
- Build System
- Tests
- Other (list it)

### CODE REVIEW

**Comment by cs5055** [comment]
Correction to  README.french.md.

File: architecture/live-queries.md [hunk 1]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
 ## Benefits of this approach
 Hasura makes live-queries easy and accessible. The notion of queries is easily extended to live-queries without any extra effort on the part of the developer using GraphQL queries. This is the most important thing for us.
 
-1. Expressive/featureful live queries with full support for Postgres operators/aggregations/views/functions etc
-1. Predictable performance
-1. Vertical & Horizontal scaling
-1. Works on all cloud/database vendors
+1. Expressive/featureful live queries with full support for Postgres operators/aggregations/views/functions etc.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed

File: architecture/live-queries.md [hunk 2]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
...
+1. Predictable performance.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed.

File: architecture/live-queries.md [hunk 3]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
...
+1. Vertical & Horizontal scaling.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed.

File: architecture/live-queries.md [hunk 4]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
...
+1. Works on all cloud/database vendors.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed.

File: architecture/live-queries.md [hunk 5]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
...
 
 ## Future work:
 Reduce load on Postgres by:
-1. Mapping events to active live queries
-1. Incremental computation of result set
+1. Mapping events to active live queries.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed

File: architecture/live-queries.md [hunk 6]
```diff
@@ -142,12 +142,12 @@ Testing scalability & reliability for live-queries with websockets has been a ch
...
+1. Incremental computation of result set.
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Not needed

**Comment by marionschleifer** [comment]
@cs5055 The conversations are marked as resolved but I don't see any of the requested changes implemented?

**Comment by cs5055** [comment]
Should the future work section be removed?

**Comment by marionschleifer** [comment]
I'm not really sure what you mean. I made comments on some changes, saying they are not needed. Could you revert these changes please. They show under resolved conversations above.

**Comment by cs5055** [comment]
Is it okay now?

File: architecture/live-queries.md [hunk 7]
```diff
@@ -141,13 +141,3 @@ Testing scalability & reliability for live-queries with websockets has been a ch
 
 ## Benefits of this approach
 Hasura makes live-queries easy and accessible. The notion of queries is easily extended to live-queries without any extra effort on the part of the developer using GraphQL queries. This is the most important thing for us.
-
-1. Expressive/featureful live queries with full support for Postgres operators/aggregations/views/functions etc
```
Comment (marionschleifer) [inline architecture/live-queries.md]: I didn't mean to delete this section. Sorry for the misunderstanding. You added a full stop to each of the points, and I meant that these full stops are not necessary. So basically this part should keep unchanged 🙂

File: architecture/live-queries.md [hunk 8]
```diff
@@ -100,7 +100,7 @@ GraphQL query → GraphQL AST → Internal AST with authorization rules → SQL
 
 #### Idea #3: Batch multiple live-queries into one SQL query
 
-With only ideas #1, #2 implemented we would still result in the situation where a 100k connected clients could result in a proportional load of 100k postgres queries to fetch the latest data (let’s say if 100k updates happen, 1 update relevant to each client).
+With only ideas [#1](https://github.com/hasura/graphql-engine/blob/master/architecture/live-queries.md#idea-1-compile-a-graphql-query-to-a-single-sql-query), [#2](https://github.com/hasura/graphql-engine/blob/master/architecture/live-queries.md#idea-2-make-authorization-declarative) implemented we would still result in the situation where a 100k connected clients could result in a proportional load of 100k postgres queries to fetch the latest data (let’s say if 100k updates happen, 1 update relevant to each client).
```
Comment (marionschleifer) [inline architecture/live-queries.md]: Can we use relative links here for just inside the document (like it's done in the table of contents, e.g. here: https://github.com/hasura/graphql-engine/blob/master/README.md.

File: architecture/live-queries.md (hunk 7 above)
Comment (cs5055) [reply architecture/live-queries.md]: Sorry i will add it again.

File: architecture/live-queries.md (hunk 8 above)
Comment (cs5055) [reply architecture/live-queries.md]: yeah sure i will update you.

**Comment by cs5055** [comment]
Is the update okay? Sorry for the delay

**Comment by cs5055** [comment]
Thank you for your support. Great working with you @marionschleifer.

**Pull request merged**